app/main.py

```diff
@@ -1 +1,58 @@
-#  write your code here
+from typing import Any
+
+
+class Animal:
+    def __init__(self,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line